hw32/js/index.js

```diff
@@ -0,0 +1,62 @@
+import { Validator } from './validator.js';
+import { signupFormConfig } from './form-configs.js';
+
+let form = document.querySelector('.signup-form');
+
```

Comment:
Please refactor follow code. It's my quick-hand code and it should be refactored to get rid of duplicates